src/ResourceManager/DataFactories/Commands.DataFactories.Test/packages.config

```diff
@@ -17,6 +17,7 @@
   <package id="Microsoft.Data.Edm" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.Data.OData" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net45" />
+  <package id="Microsoft.DataTransfer.Gateway.Encryption" version="1.7.5798-preview-001" targetFramework="net45" />
```

Comment:
Where do you need functionality from this package? this PR does not seems to add any. Please remove it.